core.py

```diff
@@ -72,7 +69,12 @@ def loadModule(self, sysconfig, args, **kwargs):
                 self.entryPoints[modName]["displayName"] = mod[5:]
 
     def getAvailableModules(self):
-        return self.modules
+        print('Available Modules (include system utilities)')
+        print('====================================')
+        for m in self.modules:
+            print(m)
```

Comment:
If you are printing module names rather than returning them then the method name should be different. Do we really need to print them in the core?

Reply:
I can revert these changes back.

Reply:
I will have more comments once I read all the code, which won't happen tonight.